[2022-02-02 14:08:20,883] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_for_taxi_zone.download_dataset_task scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2022-02-02 14:08:20,895] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_for_taxi_zone.download_dataset_task scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2022-02-02 14:08:20,895] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 14:08:20,895] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-02 14:08:20,895] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 14:08:20,912] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-01-01 00:00:00+00:00
[2022-02-02 14:08:20,914] {standard_task_runner.py:52} INFO - Started process 4231 to run task
[2022-02-02 14:08:20,920] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_for_taxi_zone', 'download_dataset_task', 'scheduled__2022-01-01T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/ingest_taxi_zone_data.py', '--cfg-path', '/tmp/tmpb7lim351', '--error-file', '/tmp/tmpzarq76rd']
[2022-02-02 14:08:20,921] {standard_task_runner.py:77} INFO - Job 154: Subtask download_dataset_task
[2022-02-02 14:08:20,978] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_for_taxi_zone.download_dataset_task scheduled__2022-01-01T00:00:00+00:00 [running]> on host 883ba1f1f430
[2022-02-02 14:08:21,037] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_for_taxi_zone
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-01T00:00:00+00:00
[2022-02-02 14:08:21,038] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 14:08:21,038] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sSLf https://s3.amazonaws.com/nyc-tlc/misc/taxi+_zone_lookup.csv > /opt/***/taxi+_zone_lookup.csv']
[2022-02-02 14:08:21,044] {subprocess.py:85} INFO - Output:
[2022-02-02 14:08:21,725] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 14:08:21,754] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_for_taxi_zone, task_id=download_dataset_task, execution_date=20220101T000000, start_date=20220202T140820, end_date=20220202T140821
[2022-02-02 14:08:21,779] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 14:08:21,812] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
